### PR TITLE
add primeui v3.0.0

### PR DESCRIPTION
**Pull request by maruilian11**
**git repo url:** https://github.com/primefaces/primeui-distribution
**Watch 1
  Star 4
 Fork 4**
@Amomo could u help me check this PR for #6812 ? thank you!
- [x] Not found on cdnjs repo
- [x] No already exist issue and PR
- [x] Notable popularity  cc https://github.com/primefaces/primeui https://github.com/cdnjs/cdnjs/issues/6812#issuecomment-176288240
- [x] Project has public repository on famous online hosting platform (or been hosted on npm) 
- [x] Has valid tags for each versions (for git auto-update) 
- [x] Auto-update setup
- [x] Auto-update target/source is valid.
- [x] Auto-update filemap is correct

### CODE REVIEW

File: ajax/libs/primeui/package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    "ui",
+    "web-components"
+  ],
+  "author": "PrimeTek",
+  "license": "Apache-2.0",
+  "version": "3.0.0",
+  "filename": "primeui-min.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/primefaces/primeui-distribution.git"
```
Comment (PeterDaveHello) [inline ajax/libs/primeui/package.json]: Should use `https://github.com/primefaces/primeui` here.

File: ajax/libs/primeui/package.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  },
+  "homepage": "https://github.com/primefaces/primeui-distribution",
```
Comment (PeterDaveHello) [inline ajax/libs/primeui/package.json]: Should use `http://www.primefaces.org/primeui` here.

**Comment by maruilian11** [comment]
@Amomo i have changed `homepage` and repo url to https://github.com/primefaces/primeui, please help me check this PR, thanks.

**Comment by maruilian11** [comment]
@Amomo i have changed `homepage` and repo url, please help me check this PR, thanks.

**Pull request merged**